modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc

```diff
@@ -11,36 +11,59 @@
+It allows Microsoft Office users to collaboratively work with Office documents in ownCloud in the browser, by connecting ownCloud with your Microsoft Office Online Server or Office 365 (cloud) via {wopi-protocol-url}[the WOPI protocol].
+
+ - *Microsoft Office Online Server:* To use it, you need to have a running Microsoft Office Online Server in your data center.
+ - *Office 365 (cloud):* To use it, you need an active Microsoft 365 subscription. 
 
 [NOTE]
 ====
 Please bear in mind:
 
 * WOPI is only available for ownCloud enterprise. It _is not available_ in the community version.
 * Out-of-the box only the on-premise version of Microsoft Office Online Server is supported.
+* If you want to integrate the {office365-url}[Office 365 (cloud)] version of Microsoft Office Online, you need to {oc-contact-url}[get in touch with us].
 * This app requires at minimum ownCloud Version 10.1 and php 7.1.
 ====
 
-TIP: If you want to integrate the {office365-url}[Office 365 (cloud)] version of Microsoft Office Online, you need to {oc-contact-url}[get in touch with us].
-
 == Preparing the Environment
-You need an {office-online-server-url}[Office Online Server] installed.
+You need an {office-online-server-url}[Office Online Server] installed or an active Microsoft 365 subscription.
 
-All involved servers (OfficeOnline Server and the ownCloud server) need to be accessible by HTTPS with valid certificates.
+All involved servers (Office Online Server and the ownCloud server) need to be accessible by HTTPS with valid certificates.
 
 == Configuring the WOPI App in ownCloud
 
 To configure the WOPI app in your ownCloud installation, add the following configuration to `config/config.php`, and adjust it based on the details of your setup:
 
 [source,php]
 ----
-'wopi.token.key' => 'replace-with-your-own-random-string',
-'wopi.office-online.server' => 'https://your.office.online.server.tld',
+# WOPI token:
+# For Office Online Server: Replace the token with your own random string
+# For Office 365 (cloud): {oc-contact-url}[Request the string from us] (has to match the `O365_PROXY_SHARED_KEY` configuration of the O365 proxy)
+'wopi.token.key' => 'REPLACE_WITH_WOPI_TOKEN_KEY'
+
+# Office server URL
+# For Office Online Server: Enter your your.office.online.server.tld
+# For Office 365 (cloud):
+# Upstream url to Microsoft O365. Microsoft will only accept connections from registered ownCloud domains.
+# For Office 365 quality assurance upstream URL: https://ffc-onenote.officeapps.live.com/hosting/discovery
+# For Office 365 production upstream URL: https://onenote.officeapps.live.com/hosting/discovery
+'wopi.office-online.server' => 'https://the.office.online.server.tld',
+
+# Proxy URL
+# Only for Office 365 (cloud), not needed for Office Online Server
+# URL of the O365 proxy instance, the path `/wopi/files/` is required.
+'wopi.proxy.url' => 'https://wopi.owncloud.com/wopi/files/',
```

Comment:
curl https://wopi.owncloud.com/wopi/files/
curl: (6) Could not resolve host: wopi.owncloud.com

??? should that work?